docs/getting-started.md

```diff
@@ -0,0 +1,316 @@
+If you don't know what Story Formats are, they essentially decide a lot of the syntax, look, and behavior. Twinery itself is kindof an 'engine' for these formats.  
+![Twinery Story Formats](images/twinery-storyformats.png)  
+These are the story formats the Twinery has (in the Change Format section).  
+As it says, the default is *Harlowe 3*.  
+So, if you want to use Harlowe, then write in the above:
+(Note: the versions here are slightly out of date of the ones in the picture, because Twinery ships with the latest versions, while TweeGo has out of date versions. Updating them is easily done, and will probably be in another section (TODO))
+```json
+	"format": "Harlowe",
+	"format-version": "3.1.0"
+````
+
+If you want to use SugarCube (T3LT has the most support for it!):  
+
+```json
+	"format": "SugarCube",
+	"format-version": "2.30.0"
+```
+Chapbook:
+```json
+	"format": "Chapbook",
+	"format-version": "1.0.0"
+```
+
+TweeGo also has support for other formats, and they can be added to it as well.  
+  
+Once you have that done, simply save the file. We'll be going with SugarCube-2 for this tutorial, but you should not have too many issues.  
+Once saved, you may see a popup in the corner of the VSCode window for "Storyformat set to sugarcube-2". This is just T3LT informing us that it now knows what format we're using. Before, with our singular Twee file with only a very simple passage, it couldn't know what format we wanted to use.  
+If you hover over the notification, it will have an X button so you can close it :)
+  
+Now, we could start writing more complex story now, but we should probably install TweeGo so we can actually build our story!
+
+## TweeGo
+TweeGo is an essential part of getting this all to work. You could even just use it with any text editor you want (though T3LT is currently VSCode only), but this guide doesn't cover that (though, it wouldn't be too hard to add if you wish).  
+As mentioned previously, TweeGo is by the same person who made the common SugarCube story format, and is available for download from: https://www.motoslave.net/tweego/  
+Scroll down to the downloads section and download the right zip file for whatever operating system you are on. If you're confused by the x64 (64-bit) and x86 (32-bit), you probably have a 64-bit computer in this day and age, and so get that one.  
+(If you like his software, consider donating through his Patreon!)  
+  
+Once you have it downloaded, you have to extract it somewhere. Running programs from a zip can.. sometimes work, but can also have issues, and is needed for how we're going to use it. You can usually just go to the folder the file was downloaded into, right click it, and hit some button that says Extract.  
+What I do is move the files it has (the `tweego` executable and the `storyformats` folder) and move them into the projects folder under their own folder, ex:
+```
```

Comment:
`
```plaintext`

Reply:
I'm not sure what this changes?

Reply:
everything, think of Rust strictness and obey.